userauth_test.go

```diff
@@ -141,12 +142,13 @@ func TestUserTokenAuthenticator_ValidateSignature_RSA(t *testing.T) {
 	// Test that a user can access their own bucket
 	r.URL.Path = "/dummy/"
 	s3signer.SignV4(*r, "username", "testpass", "", "us-east-1")
-	assert.Nil(t, a.Authenticate(r))
+	_, err = a.Authenticate(r)
+	assert.Nil(t, err)
```

Comment:
maybe add a check to ensure that the claims contains the relevant data

Reply:
Do you think the changes are sufficient?